### PR TITLE
New Command Traitor Item: AI Laser Upgrade Modules

### PR DESCRIPTION
**Pull request by Wisemonster**
[GAME OBJECTS] [FEATURE] [INPUT WANTED] [EXPERIMENTAL]

## About the PR 
This pr adds a new item for traitorous heads of staff, an upgrade module that lets the AI use a laser ability from their cameras. The laser module shoots a 20 damage phaser shot, with a 10 second cool down, that costs 6 tc.

## Why's this needed? 
AI ability upgrade modules have been in the code for a while now, and I figured this ability wouldn't really fit as anything other than a traitor item. Plus gives traitor members of command more control of how a rogue AI can help them.

## Changelog

```changelog
(u)Wisemonster
(*)Adds a new command traitor item: An AI laser upgrade module that costs 6 tc.

### CODE REVIEW

File: code/datums/syndicate_buylist.dm [hunk 1]
```diff
@@ -979,6 +979,24 @@ This is basically useless for anyone but miners.
 	not_in_crates = 1
 	can_buy = UPLINK_TRAITOR
 
+/datum/syndicate_buylist/traitor/ai_taser
+	name = "AI Camera Taser Module"
+	item = /obj/item/aiModule/ability_expansion/taser
+	cost = 6
+	vr_allowed = 0
```
Comment (Zonespace27) [inline code/datums/syndicate_buylist.dm:986]: true/false defines

File: code/datums/syndicate_buylist.dm [hunk 2]
```diff
@@ -979,6 +979,24 @@ This is basically useless for anyone but miners.
...
+	desc = "An AI module that upgrades any AI connected to the installed law rack access to the tasers installed in the cameras."
+	job = list("Captain", "Head of Personnel", "Research Director", "Medical Director", "Chief Engineer")
+	can_buy = UPLINK_TRAITOR | UPLINK_SPY_THIEF
+
+/datum/syndicate_buylist/traitor/ai_laser
+	name = "AI Camera Laser Module"
+	item = /obj/item/aiModule/ability_expansion/laser
+	cost = 7
+	vr_allowed = 0
```
Comment (Zonespace27) [inline code/datums/syndicate_buylist.dm:995]: ditto

**Review by Tarmunora — CHANGES_REQUESTED** [review]
I'd drop the taser module and stick with just the laser module @6TC, and maybe take it out of spy/crate pools - not too set either way on that though

**Review by Tarmunora — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]
sure let's try it

**Pull request merged**